launcher/src/main/java/org/apache/spark/launcher/CommandBuilderUtils.java

```diff
@@ -260,15 +260,14 @@ static String quoteForBatchScript(String arg) {
         quoted.append('"');
         break;
 
-      case '=':
```

Comment:
I've run `SparkLauncherSuite` on Windows and it's OK.
If double-quotation is parsed properly, `=` in double-quotation is not need to be escaped.